samcli/local/docker/lambda_container.py

```diff
@@ -31,7 +32,10 @@ class LambdaContainer(Container):
     # This is the dictionary that represents where the debugger_path arg is mounted in docker to as readonly.
     _DEBUGGER_VOLUME_MOUNT = {"bind": _DEBUGGER_VOLUME_MOUNT_PATH, "mode": "ro"}
 
-    def __init__(
+    # Additional volumes Mount path for files in docker
+    _VOLUME_MOUNT_PATH = "/tmp/lambci_volumes"
```

Comment:
Why do you need the additional directory? What do you think about mounting directly under /tmp instead?

Reply:
Additional directory is just for better folders structure. `/tmp/` I found might be pretty common place for this purpose. Similar to `/tmp/lambci_debug_files` that is mounted for all debugger files. Please let me know if you would prefer to have these files inside `/tmp/`.

Reply:
Makes sense. To Alex's comments, can you make this folder more descriptive like `/tmp/additional_volumes`?